munemo/munemo.go

```diff
@@ -14,14 +15,23 @@ func Munemo(id int) string {
 	return m.string()
 }
 
+func UnMunemo(s string) int {
```

Comment:
If this can fail, lets add an error return.

Reply:
yeah. I wasn't sure of the idiom. This code seems mostly OO oriented. Creates an object, then calls methods on said object. No real returns, per se. I'll try to mock up something